client/my-sites/theme/controller.jsx

```diff
@@ -30,31 +30,33 @@ export function fetchThemeDetailsData( context, next ) {
 		return next();
 	}
 
+	const siteId = 'wpcom';
```

Comment:
I'm a little wary of this because it gives the impression in the code where it used, on first glance, that an actual site ID is being passed, which would never be the case.